python_modules/dagster/dagster/_core/definitions/asset_reconciliation_sensor.py

```diff
@@ -848,7 +975,12 @@ def reconcile(
     instance: "DagsterInstance",
     cursor: AssetReconciliationCursor,
     run_tags: Optional[Mapping[str, str]],
-):
+) -> Tuple[
+    Sequence[RunRequest],
+    AssetReconciliationCursor,
+    Mapping[AssetKeyPartitionKey, AbstractSet[AutoMaterializeReason]],
+    Mapping[AssetKeyPartitionKey, AbstractSet[AutoMaterializeSkipReason]],
```

Comment:
Yeah that's fair -- originally did this to make it easier to write tests because it was annoying to manually construct those big NamedTuples, but it's a bit nicer now